src/realmGeol.ttl

```diff
@@ -57,307 +288,1624 @@ sorelsp:hasUpperBoundary rdf:type owl:ObjectProperty .
-                                 [ rdf:type owl:Restriction ;
-                                   owl:onProperty sorelsc:containsMaterial ;
-                                   owl:allValuesFrom somarock:Rock
-                                 ] ;
-                 rdfs:label "lithosphere"@en .
+                   rdfs:subClassOf soreag:GeosphereLayer ,
+                                   [ rdf:type owl:Restriction ;
+                                     owl:onProperty sorelsc:containsMaterial ;
+                                     owl:allValuesFrom somarock:Rock
+                                   ] ;
+                   rdfs:label "lithosphere"@en .
+
+
+###  http://sweetontology.net/realmGeol/LithostratigraphicUnit
+soreag:LithostratigraphicUnit rdf:type owl:Class ;
+                              rdfs:subClassOf soreag:StratigraphicUnit ;
+                              dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                              dcterms:source <http://www.stratigraphy.org/index.php/ics-stratigraphicguide> ;
+                              skos:definition "Lithostratigraphic units are bodies of rocks, bedded or unbedded, that are defined and characterized on the basis of their lithologic properties and their stratigraphic relations. Lithostratigraphic units are the basic units of geologic mapping. (ICS, Online, December 2019)"@en ;
+                              skos:prefLabel "lithostratigraphic unit"@en .
 
 
 ###  http://sweetontology.net/realmGeol/Lopolith
 soreag:Lopolith rdf:type owl:Class ;
-              rdfs:subClassOf soreag:PlutonicStructure ;
-              rdfs:label "lopolith"@en .
+                rdfs:subClassOf soreag:PlutonicStructure ;
+                rdfs:label "lopolith"@en .
 
 
 ###  http://sweetontology.net/realmGeol/LowerMantle
 soreag:LowerMantle rdf:type owl:Class ;
-                 rdfs:subClassOf soreag:Mantle ;
-                 rdfs:label "lower mantle"@en .
+                   rdfs:subClassOf soreag:Mantle ;
+                   rdfs:label "lower mantle"@en .
+
+
+###  http://sweetontology.net/realmGeol/MagnetostratigraphicPolaritySubzone
+soreag:MagnetostratigraphicPolaritySubzone rdf:type owl:Class ;
+                                           rdfs:subClassOf soreag:MagnetostratigraphicUnit ;
+                                           dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                                           dcterms:source <http://www.stratigraphy.org/index.php/ics-stratigraphicguide> ,
+                                                          <https://doi.org/10.1130/9780813774022> ;
+                                           skos:definition "Polarity Zones may be subdivided into Polarity Suzones. The corresponding interval of geological time during which the rocks in the Polarity Subzone were deposited is referred to as its associated Polarity Subchron. (Salvador, 2013; ICS, Online, December, 2019)"@en ;
```

Comment:
skos:definition is not a definition.

Definition: A polarity subzone is a part of a Polarity zone, distinguished by ??? its remnant magnetization characteristics???